packages/beeq/src/components/drawer/_storybook/bq-drawer.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit-html';
+
+import mdx from './bq-drawer.mdx';
+import { DRAWER_PLACEMENT } from '../bq-drawer.types';
+
+const meta: Meta = {
+  title: 'Components/Drawer',
+  component: 'bq-drawer',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    open: { control: 'boolean' },
+    placement: { control: 'select', options: [...DRAWER_PLACEMENT] },
+    'disable-close-click-outside': { control: 'boolean' },
+    'disable-close-esc-keydown': { control: 'boolean' },
+    'enable-backdrop': { control: 'boolean' },
+    // Events
+    bqShow: { action: 'bqOpen' },
+    bqHide: { action: 'bqClose' },
+    bqAfterOpen: { action: 'bqAfterOpen' },
+    bqAfterClose: { action: 'bqAfterClose' },
+  },
+  args: {
+    open: false,
+    placement: 'left',
+    'disable-close-click-outside': true,
+    'disable-close-esc-keydown': true,
+    'enable-backdrop': false,
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => {
+  const handleOpenDrawer = async () => {
+    const dialogElem = document.querySelector('bq-drawer');
+    await dialogElem.show();
+  };
+
+  return html`
+    <bq-button @bqClick=${handleOpenDrawer}>Open Drawer</bq-button>
+    <bq-drawer
+      ?open=${args.open}
+      placement=${args.placement}
+      disable-close-click-outside=${args['disable-close-click-outside']}
+      disable-close-esc-keydown=${args['disable-close-esc-keydown']}
+      enable-backdrop=${args['enable-backdrop']}
+      @bqCancel=${args.bqCancel}
```

Comment:
I don't think we have `bqCancel` emitted as custom event from the component, right?

Reply:
Yes, we don't. I'll take it out.